src/CitizenProject/CitizenProjectCreationCommandHandler.php

```diff
@@ -29,6 +32,14 @@ public function handle(CitizenProjectCreationCommand $command): void
 
         $command->setCitizenProject($citizenProject);
 
+        $committeeSupports = $citizenProject->getCommitteeSupports()->toArray();
```

Comment:
This should be part of the command logic, having many instance types within the same collection in the first place feels bad from a model POV.

Reply:
✔️